src/transformers/generation_flax_utils.py

```diff
@@ -1,28 +0,0 @@
-# coding=utf-8
```

Comment:
hehe these `generation_xxx_utils.py` were already removed in https://github.com/huggingface/transformers/pull/29782 😎

Reply:
yep!